packages/components/src/Select.tsx

```diff
@@ -0,0 +1,264 @@
+ *
+ * Also (via findValue) allows passing value as the selected value instead of the selected option.
+ * For example:
+ * const options = [{ value: 'foo' }, { value: 'bar' }]
+ * h(BaseSelect, { options, value: 'foo' })
+ * instead of:
+ * const options = [{ value: 'foo' }, { value: 'bar' }]
+ * h(BaseSelect, { options, value: options[0] })
+ */
+export const BaseSelect = <
+  Value,
+  Option extends { value: Value; label?: string | undefined },
+  IsMulti extends boolean,
+  Group extends RSelectGroupBase<Option>
+>(
+  props: BaseSelectProps<Value, Option, IsMulti, Group>
+): ReactNode => {
+  const { findValue, id, value, ...otherProps } = props;
+
+  const theme = useThemeFromContext();
+
+  const newValue: RSelectPropsValue<Option> = otherProps.isMulti
+    ? _.compact(_.map(findValue, value as Value[]))
+    : findValue(value as Value);
+  const myId = useUniqueId();
+  const inputId = id || myId;
+
+  const formatGroupLabel = (group: { label?: string }): ReactNode => {
+    return (
+      <div
+        style={{
+          color: theme.colors.dark(),
+          fontSize: 14,
+          height: 30,
+          fontWeight: 600,
+          borderBottom: `1px solid ${theme.colors.dark(0.25)}`,
+        }}
+      >
+        {group.label}
+      </div>
+    );
+  };
+
+  const ParameterizedRSelect = RSelect as typeof RSelect<Option, IsMulti, Group>;
+
+  return (
+    <ParameterizedRSelect
+      {..._.merge(
+        {
+          inputId,
+          ...getCommonSelectProps(theme),
+          getOptionLabel: ({ value, label }) => label || value.toString(),
+          value: newValue || null, // need null instead of undefined to clear the select
+          formatGroupLabel,
+        },
+        otherProps
+      )}
+    />
+  );
+};
+
+export type SelectProps<
+  Value,
+  IsMulti extends boolean,
+  Option extends { value: Value; label?: string | undefined }
+> = Omit<BaseSelectProps<Value, Option, IsMulti, never>, 'findValue' | 'options'> & {
+  options: (Value | Option)[];
+};
+
+/**
+ * @param props - see {@link https://react-select.com/props#select-props}
+ * @param props.value - a member of options
+ * @param props.options - can be of any type; if objects, they should each contain a value and label, unless defining getOptionLabel
+ * @param props.onChange - The function to call when a user makes a selection
+ * @param [props.getOptionLabel] - a function to custom style the options
+ * @param [props.id] - The HTML ID to give the form element
+ * @param [props.isClearable] - whether the select can be cleared
+ * @param [props.isDisabled] - whether the select is disabled
+ * @param [props.isSearchable] - whether the select can be cleared
+ * @param [props.isMulti] - whether the select is multiselect or not
+ * @param [props.menuPlacement] - determines where the menu is placed
+ * @param [props.placeholder] - The placeholder value for the select
+ * @param [props.styles] - custom styling for the select
+ */
+export const Select = <
```

Comment:
We will need unit tests for Select.tsx now that it is in this package... You can see other reusable components for an appropriate level of testing.  Currently this PR has Select.tsx at 0% coverage, which needs to be 80% or better.